test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -792,6 +813,7 @@ def deprecation_schema(for_collection):
         result = All(
             result,
             partial(check_removal_version,
+            require_only_one(['alternative', 'alternatives']),
```

Comment:
This either needs to go a line up, or several down.

Reply:
done, thanks for catching that